controllers/replication-controller/dellcsireplicationgroup_controller.go

```diff
@@ -469,7 +506,8 @@ func makeVolSnapContent(snapName, volumeName string, snapRef v1.ObjectReference,
 }
 
 // SetupWithManager start using reconciler by creating new controller managed by provided manager
-func (r *ReplicationGroupReconciler) SetupWithManager(mgr ctrl.Manager, limiter ratelimiter.RateLimiter, maxReconcilers int) error {
+func (r *ReplicationGroupReconciler) SetupWithManager(mgr ctrl.Manager, limiter ratelimiter.RateLimiter, maxReconcilers int, disablePVCRemap bool) error {
+	r.DisablePVCRemap = disablePVCRemap
```

Comment:
**issue:** We shouldn't need to set this here. https://github.com/dell/csm-replication/pull/147/files#diff-dfc3fcfcfdeda946477dcb3b37d036edc40e8b902af04a9eb6cf3c27ccd7e36fR248 already takes care of setting the variable.